packages/cfd/src/_common/__tests__/utility.ts

```diff
@@ -0,0 +1,15 @@
+import { template } from '../utility';
+
+describe('Utility', () => {
+    describe('.template()', () => {
+        it('works as expected', () => {
```

Comment:
test case description is vague.
```suggestion
        it('should return correct template format, () => {
```